trin-core/src/portalnet/overlay_service.rs

```diff
@@ -54,6 +59,12 @@ pub const FIND_CONTENT_MAX_NODES: usize = 32;
 /// With even distribution assumptions, 2**17 is enough to put each node (estimating 100k nodes,
 /// which is more than 10x the ethereum mainnet node count) into a unique bucket by the 17th bucket index.
 const EXPECTED_NON_EMPTY_BUCKETS: usize = 17;
+/// Timeout for FINDNODES queries in seconds.
+pub const QUERY_TIMEOUT: u64 = 60;
```

Comment:
If these two constants are specific to `FINDNODES` - I'd prefix the const name with `FIND_NODES_*`